guvnor-bom/pom.xml

```diff
@@ -164,6 +164,12 @@
       </dependency>
       <dependency>
         <groupId>org.guvnor</groupId>
+        <artifactId>guvnor-ala-build-maven</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>tests</classifier>
```

Comment:
There are two options for test jars:
 1. `<classifier>tests</classifier>`
 2. `<scope>test</scope>`

We are mixing both in different POMs, so it probably does not matter which one gets used.